subsys/bluetooth/host/scan.c

```diff
@@ -716,22 +717,30 @@ void bt_hci_le_per_adv_sync_established(struct net_buf *buf)
 			      BT_PER_ADV_SYNC_SYNCING_USE_LIST) &&
 	     ((pending_per_adv_sync->sid != evt->sid) ||
 	      bt_addr_le_cmp(&pending_per_adv_sync->addr, &evt->adv_addr)))) {
-		struct bt_le_per_adv_sync_term_info term_info;
-
 		BT_ERR("Unexpected per adv sync established event");
+		/* Request terminate of pending periodic advertising in controller */
 		per_adv_sync_terminate(sys_le16_to_cpu(evt->handle));
 
+		unexpected_evt = true;
+	} else {
+		unexpected_evt = false;
+	}
+
+	if ((unexpected_evt == false && evt->status != BT_HCI_ERR_SUCCESS) ||
```

Comment:
Variables of type `bool` are valid Boolean expressions in themselves, so explicit comparisons with `true` or `false` seem redundant and don’t improve readability, at least if the variable is appropriately named. So I’d prefer ‘!unexpected_evt’ etc. It even reads as more natural English “not unexpected event”

Reply:
Thanks, I've forgotten to correct it here. Now it is done.